prime-router/docs/design/proposals/0024-original-message-passthrough.md

```diff
@@ -0,0 +1,135 @@
+# Proposal for Original Message Passthrough
+
+## Background
+
+In order to meet data integrity requirements, a sender of HL7v2 has requested a feature in the Universal Pipeline to
+preserve the original unaltered message and convey it to receivers with no alterations, including preserving the
+original file name.
+
+## Assumptions
+
+This design is strictly for an HL7v2 sender and does not apply to a FHIR sender. Likewise, this does not apply to a FHIR
```

Comment:
I would change this to "this design is for non-FHIR messages". It shouldn't matter if its HL7v2 or CSV or whatever, we should be able to stick it into the FHIR bundle (or pull it from azure).

Reply:
This limitation is currently inherent to the Universal Pipeline not supporting CSV; to build this feature to support CSV would currently require extending the legacy pipeline as well.